src/app/static/src/js/components/documents/documentPage.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <div>
+        <div class="row">
+            <div class="col-8" v-if="canManage">
+                <label class="font-weight-bold">Manage project docs</label>
+                <refresh-documents @refreshed="getDocs"></refresh-documents>
+            </div>
+        </div>
+        <document-list :docs="docs"></document-list>
+    </div>
+</template>
+
+<script>
+    import documentList from "./documentList";
+    import {api} from "../../utils/api";
+    import refreshDocuments from "./refreshDocuments";
+
+    export default {
+        name: "document-app",
+        props: ["canManage"],
+        data() {
+            return {
+                docs: []
+            }
+        },
+        components: {
+            documentList,
+            refreshDocuments
+        },
+        methods: {
+            getDocs() {
+                api.get("/documents/")
+                    .then(({data}) => {
+                        this.docs = data.data;
+                    })
+            }
+        },
+        created() {
+            this.getDocs()
```

Comment:
As we're now fetching documents asynchronously, can remove them from the viewmodel, but I figured this changeset was big enough as is, so I'll do that on a further branch.